src/types/observation.js

```diff
@@ -19,5 +19,4 @@ export interface Observation {
   observedBy: string;
   media: ObservationMedia[];
   icon: any;
-  categoryId?: string;
```

Comment:
Think we actually need this for the Category name in ObservationCell and ObservationDetailView